examples/resource_lacework_integration_ecr/iam_role/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "integration_name" {
+  type    = string
+  default = "Amazon Elastic Container Registry Example"
+}
+variable "role_arn" {
+  type      = string
+  sensitive = true
+  default   = "arn:aws:iam::1234567890:role/lacework_iam_example_role"
+}
+variable "external_id" {
+  type      = string
+  sensitive = true
+  default   = "12345"
+}
+variable "registry_domain" {
+  type      = string
+  default   = "YourAWSAccount.dkr.ecr.YourRegion.amazonaws.com"
+}
+variable "non_os_packages" {
```

Comment:
nit: same comment as other pr. Let's rename this variable to something more clear like `non_os_package_support`